templates/zerver/help/include/change-zulip-config-file.md

```diff
@@ -0,0 +1,9 @@
+Next, on your {{ settings_html|safe }}, create a bot for
+{{ integration_display_name }}.
+
+Next, open
```

Comment:
This part caught my eye as awkward on the multiple pages where it came up. 
Maybe change one of them so they don't both say "Next,"